{"name": "test_create_obj[data2]", "status": "failed", "statusDetails": {"message": "AssertionError: Object with id ff8081818eeb2234018eec755339063c wasn't deleted", "trace": "data = {'data': {'CPU model': 'Intel Core i9DKA', 'Hard disk size': '1 TB DKA', 'price': 2805.99, 'year': 1982}, 'name': 'DKA_3'}, create_new_obj_endpoint = <endpoints.create_obj.CreateObject object at 0x000002D7E523BC20>\ndel_obj_by_id_endpoint = <endpoints.delete_obj.DeleteObject object at 0x000002D7E5288170>\n\n    @pytest.mark.parametrize(\"data\", TEST_DATA)\n    def test_create_obj(data, create_new_obj_endpoint, del_obj_by_id_endpoint):\n        create_new_obj_endpoint.create_new_obj(data)\n        create_new_obj_endpoint.check_obj_name(data['name'])\n>       create_new_obj_endpoint.del_obj()\n\ntests\\test_some_app.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <endpoints.create_obj.CreateObject object at 0x000002D7E523BC20>\n\n    @allure.step('Удаление объекта.')\n    def del_obj(self):\n        self.response = requests.delete(\n            f\"{self.url}/{self.obj_json['id']}\"\n        )\n        self.response = requests.get(\n            f\"{self.url}/{self.obj_json['id']}\")\n>       assert self.response.status_code == 403, f\"Object with id {self.obj_json['id']} wasn't deleted\"\nE       AssertionError: Object with id ff8081818eeb2234018eec755339063c wasn't deleted\n\nendpoints\\create_obj.py:32: AssertionError"}, "steps": [{"name": "Создаем новый объект.", "status": "passed", "parameters": [{"name": "body", "value": "{'name': 'DKA_3', 'data': {'year': 1982, 'price': 2805.99, 'CPU model': 'Intel Core i9DKA', 'Hard disk size': '1 TB DKA'}}"}], "start": 1713364094094, "stop": 1713364095578}, {"name": "Проверяем имя созданного объекта.", "status": "passed", "parameters": [{"name": "name", "value": "'DKA_3'"}], "start": 1713364095578, "stop": 1713364095578}, {"name": "Удаление объекта.", "status": "failed", "statusDetails": {"message": "AssertionError: Object with id ff8081818eeb2234018eec755339063c wasn't deleted\n", "trace": "  File \"C:\\Kirill_Dechko\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Kirill_Dechko\\test_api_dka\\endpoints\\create_obj.py\", line 32, in del_obj\n    assert self.response.status_code == 403, f\"Object with id {self.obj_json['id']} wasn't deleted\"\n"}, "start": 1713364095578, "stop": 1713364097949}], "parameters": [{"name": "data", "value": "{'name': 'DKA_3', 'data': {'year': 1982, 'price': 2805.99, 'CPU model': 'Intel Core i9DKA', 'Hard disk size': '1 TB DKA'}}"}], "start": 1713364094094, "stop": 1713364097949, "uuid": "c3f5bf2f-35fe-40af-82a1-0c1f16752528", "historyId": "9f21072b0029d5a98b3bb039bb787c71", "testCaseId": "446c5770a8159f9ad79dc3323a217e9b", "fullName": "tests.test_some_app#test_create_obj", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_some_app"}, {"name": "host", "value": "Kirill"}, {"name": "thread", "value": "30768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_some_app"}]}